src/jit/target.h

```diff
@@ -40,6 +40,10 @@
 /*****************************************************************************/
 
 #if defined(_TARGET_ARM_)
+// TODO: This is a hack, what's the correct solution?
+#ifdef __int64
+#undef __int64
+#endif
```

Comment:
We define __int64 to unsigned long long, this breaks the __int64(x) calls in registerarm.h . Not sure what is the correct solution here.